package.json

```diff
@@ -1,12 +1,13 @@
 {
 	"scripts": {
 		"test": "xo && ava && npm run build",
-		"build": "rollup -c",
-		"watch": "run-p --continue-on-error --silent \"build -- --watch\"",
-		"watch:sourcemap": "cross-env SOURCEMAP=true npm run watch --silent",
+		"build": "webpack",
+		"watch": "webpack --watch",
+		"watch:sourcemap": "webpack --watch --devtool eval-cheap-module-source-map",
+		"reduce-size": "babel --out-dir . --no-comments extension/*.js",
```

Comment:
Oh, I see you originally had that, why did you revert it?

https://github.com/sindresorhus/refined-github/pull/577/commits/bd8b1a6aa9cb453b3959f881f3f822ce9f6cd916